src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/RequiresDynamicCodeAttribute.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
```

Comment:
This is a copypaste from `RequiresUnreferencedCodeAttribute` where this is likely needed because the attribute is included from multiple projects. It's not necessary here at the moment.

Reply:
I was doing some tests with the CodeFixer in the analyzer space and it generates "error CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context."

Reply:
Ah I see now the Url is marked as nullable